src/components/sbb-form-field/sbb-form-field.scss

```diff
@@ -97,6 +98,11 @@
   --sbb-form-field-background-color: var(--sbb-color-white-default);
 }
 
+:host([width='collapse']) {
```

Comment:
I'm not sure whether we should change the notation to the following. It would mean that we never override a definition and it's maybe more consumer friendly. what do you think @kyubisation?
```suggestion
:host(:where(:not([width='collapse']))) {
  min-width: #{sbb.px-to-rem-build(150)};
  width: min(#{sbb.px-to-rem-build(300)}, 100%);
}
```

Reply:
Discussed with Teams. I agree with the proposal.